src/components/prescriptions/PrescritionImport.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+    <div
+        class="prescriptionBox"
+        @submit.prevent="addTest">
+        <h3 class="prescriptionBox__header">Dodaj receptę:</h3>
+        <form class="prescriptionBox__form prescriptionForm">
+            <div>
+                <div class="inputBox">
+                    <label for="nameAndSurnameDoctor">
```

Comment:
same there is no need to add label markup. As I can see there is not corrected name: 'Imie i nazwisko lekarza" and we considered to change to just "Lekarz"